src/server.ts

```diff
@@ -1,11 +1,12 @@
 import App from '@/app';
 import AuthRoute from '@routes/auth.route';
+import GroupsRoute from './routes/group.route';
```

Comment:
It's recommended to maintain consistency in route imports. Consider using '@routes/group.route' to match the pattern used for other route imports.